strawberryfields/apps/similarity.py

```diff
@@ -359,21 +488,21 @@ def prob_orbit_mc(
 
     prob = 0
 
-    for _ in range(samples):
+    for _ in range(mc_samples):
         sample = orbit_to_sample(orbit, modes)
         prob += result.state.fock_prob(sample, cutoff=photons + 1)
 
-    prob = prob * orbit_cardinality(orbit, modes) / samples
+    prob = prob * orbit_cardinality(orbit, modes) / mc_samples
 
     return prob
 
 
-def prob_event_mc(
+def _prob_event_mc(

```

Comment:
What is the motivation for making these `_prob_...` functions private? I think they can be left public (without the first underscore)

Reply:
The idea is to provide only high-level functions that are compact and to avoid clutter. We decided this in the ADR but we can reconsider this. It is possible that the user has cases where it wants to calculate just a single orbit/event probability rather than the whole feature vector.

Reply:
okay I will make them all public.